apps/web/src/components/docs/Docs.tsx

```diff
@@ -105,15 +151,38 @@ export const Docs = ({ code = '', description = '', title = '', isLoading, child
           textAlign: 'justify left',
         })}
       >
-        <div>
-          <Flex justify="space-between" align="center">
-            <TitleH1>{title}</TitleH1>
-            {actions}
-          </Flex>
-          <Text>{description}</Text>
-        </div>
+        <When truthy={!isChildDocs}>
+          <div>
+            <Flex justify="space-between" align="center">
+              <TitleH1
+                className={css({
+                  width: '99%',
+                })}
+              >
+                {title}
+              </TitleH1>
+              {actions}
+            </Flex>
+            <Text>{description}</Text>
+          </div>
+        </When>
         <Component
           components={{
+            CodeGroup: ({ children, ...props }: { children: ReactNode }) => {
+              return <div>{children}</div>;
+            },
+            ResponseExample: ({ children }: { children: ReactNode }) => {
+              return <div>{children}</div>;
+            },
+            RequestExample: ({ children }: { children: ReactNode }) => {
+              return <div>{children}</div>;
+            },
+            ParamField: ({ children }: { children: ReactNode }) => {
+              return <div>{children}</div>;
+            },
+            Expandable: ({ children }: { children: ReactNode }) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Consider a 'DefaultComponent" that we can use for each of these (and others) to centralize the implementation.‏

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Also, do we need the wrapping div?‏

Reply:
It is there to make sure it ends up as it owns block